router/manager/manager.go

```diff
@@ -15,9 +15,9 @@ import (
 )
 
 var (
-	objectStorageDestinations []string
-	asyncDestinations         []string
-	warehouseDestinations     []string
+	objectStorageDestinations = []string{"S3", "GCS", "AZURE_BLOB", "MINIO", "DIGITAL_OCEAN_SPACES"}
```

Comment:
I cannot find how these slices were initialized before

Reply:
cc @kuldeep0020

Reply:
These were hardcoded before as well, we are not reading these from any config or storage as of now.

Reply:
This is hardcoded as of now in router.go